src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -62,42 +76,25 @@ function getPlotBands(cities, city, color) {
   return [];
 }
 
-const CityColumnChart = (props) => {
```

Comment:
This diff is a dog's breakfast because I've moved `getChartConfig()` into its own function and git got confused.